Tests/CatalogItems/Rest/Get-RsRestCacheRefreshPlanHistory.Tests.ps1

```diff
@@ -61,13 +61,20 @@ Describe "Get-RsRestCacheRefreshPlanHistory" {
             $plan | Should Not BeNullOrEmpty
             $plan.LastStatus | Should Not Be 'New Scheduled Refresh Plan'
 
-            Start-Sleep -Seconds 3
+            Start-Sleep -Seconds 6
```

Comment:
I would remove this wait now that you have the polling

Reply:
Done.